components/auth/form/TermsForm.tsx

```
@@ -0,0 +1,97 @@
+
+  const handleBackClick = () => {
+    router.back();
+  };
+
+  const handleDetailInfoClick = (id: string) => {
+    router.push(`/terms/${id}`);
+  };
+
+  const handleAllConsentClick = () => {
+    setCheckedMap(() =>
+      AGREEMENTS.reduce((acc, { id }) => ({ ...acc, [id]: !isAllCheck }), {})
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)